UICatalog/Properties/launchSettings.json

```diff
@@ -1,7 +1,8 @@
 {
   "profiles": {
     "UICatalog": {
-      "commandName": "Project"
+      "commandName": "Project",
+      "commandLineArgs": "-d NetDriver"
     },
     "UICatalog --driver NetDriver": {
```

Comment:
There is already a NetDriver for the UICatalog. This is really needed?

Reply:
Reverted, this was change made in UI